File: FTAnalyser/Old Java Code/org/hardgate/familytree/comparators/NameComparator.java
/*
 * Created on 04-Jan-2005
 *
 */
package org.hardgate.familytree.comparators;

import java.util.Comparator;

import org.hardgate.familytree.core.Individual;
import org.hardgate.familytree.registrations.Registration;

/**
 * @author db
 *
 */
public class NameComparator implements Comparator<Registration> {

    public int compare (Registration r1, Registration r2) {
        Individual i1 = r1.getIndividual();
        Individual i2 = r2.getIndividual();
        return i1.compareTo(i2);
    }
}

File: FTAnalyser/Comparators/NameComparator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml;

namespace FTAnalyser
{
    public class NameComparator : IComparer<Registration>
    {
        public int Compare (Registration r1, Registration r2) {
            Individual i1 = r1.Individual;
            Individual i2 = r2.Individual;
            return i1.CompareTo(i2);
        }
    }
}
